### ISSUE TITLE
Unbale to find variable declaretion ($Bin)

### ISSUE THREAD

**Issue by targzeta** [Camelcade/Perl5-IDEA#2321]
```perl
use File::Spec::Functions qw(catfile);
use FindBin qw($Bin);
...
catfile($Bin, 'dir'); <--- Unbale to find variable declaretion. The variable is $Bin
```

Best regards,
Emanuele

**Comment by hurricup** [comment]
See https://github.com/Camelcade/Perl5-IDEA/issues/1284
Probably this gonna be fixed with #1860

**Comment by targzeta** [comment]
Ok, thank you!

Emanuele
